run-xunit-perf.cmd

```diff
@@ -0,0 +1,56 @@
+@setlocal
```

Comment:
This file seems like it would be better under the scripts folder $(projectRoot)/tests/scripts